### ISSUE TITLE
Symbolic package fails to load when python 3.10 is used

### ISSUE THREAD

**Issue by forGGe** [cbm755/octsympy#1058]
In octave console:

```
pkg install -forge symbolic
pkg load symbolic
syms x
```

Yeilds:

```
>> syms x
Symbolic pkg v2.9.0: <stdin>:23: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
Traceback (most recent call last):
  File "<stdin>", line 28, in <module>
AttributeError: '_PrintFunction' object has no attribute '__globals__'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 12, in octoutput_drv
  File "<stdin>", line 54, in octoutput
  File "<stdin>", line 55, in octoutput
AttributeError: module 'sympy' has no attribute 'compatibility'
Closing the Python communications link.

error: Python exception: AttributeError: '_PrintFunction' object has no attribute '__globals__'
    occurred in python_header import block.
    Try "sympref reset" and repeat your command?
    (consider filing an issue at https://github.com/cbm755/octsympy/issues)
error: called from
    pycall_sympy__ at line 191 column 5
    valid_sym_assumptions at line 38 column 10
    assumptions at line 82 column 7
    syms at line 97 column 13
```

**Comment by nzec** [comment]
Install SymPy 1.5.1 using `pip install --user sympy==1.5.1`

**Comment by cbm755** [comment]
I think this is now fixed/worked around but I'll leave it open until we have #1069 fixed.  And maybe another look at the pretty printing monkey patching would be good.

For released versions, @nzec's workaround is currently a good option.

**Comment by cbm755** [comment]
At least doctests run and pass on Octave 7.1's Docker images, which are based on Python 3.10.  Fixed by  #1123 which will be merged soon.
